openpgp.d.ts

```diff
@@ -531,11 +529,11 @@ export namespace stream {
 
 /* ############## v5 GENERAL #################### */
 
-export interface UserId { name?: string; email?: string; }
+export interface UserId { name?: string; email?: string; comment?: string; }
 export interface SessionKey { data: Uint8Array; algorithm: string; }
 
 
-interface EncryptOptions {
+export interface EncryptOptions {
```

Comment:
I think this one should be a declaration as well - but it seems like it isn't necessary to explicitly `declare` `interface`s, so I would just revert this one and change all the other `declare interface`s to `interface`

Reply:
Sorry, what I meant was that I think this one probably makes more sense unexported as well

Reply:
I think it may be useful to export it and allow users to instantiate and type-check objects before passing them to the function?
Something like:
```
let options : DecryptOptions;
options.message = ...;
```
I have no idea if people use TS in this way though 😄